cmd/dt/carvelize.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware-labs/distribution-tooling-for-helm/chartutils"
+	carvel "github.com/vmware-labs/distribution-tooling-for-helm/chartutils/carvel"
```

Comment:
This is what I mentioned in carvel.go, If you move the dir to the top of the repo and use "package carvel", this would look like:

```
import (
    "github.com/vmware-labs/distribution-tooling-for-helm/carvel"
)
```